ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -281,6 +282,8 @@ export const testBoxInBox = (): Promise<boolean> =>
             type: WhisperComponentType.Box,
             alignment: JustifyContent.Center,
             direction: Direction.Horizontal,
+            // eslint-disable-next-line no-underscore-dangle
```

Comment:
Do you still need this lint rule override?

Reply:
> Do you still need this lint rule override?

That's right, we won't need that lint rule since there will be no more underscores :)